config/routes.rb

```diff
@@ -1,6 +1,13 @@
 Rails.application.routes.draw do
+  devise_for :user
+  resources :transract_groups
+  resources :groups, only: [:index, :create, :new, :destroy] do
+  resources :transracts
+  end
+  resources :users
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
   # root "articles#index"
+  root "users#index"
```

Comment:
- [ ] Kindly make this the `groups/index` as the default route so that when the user is authenticated he needs to be redirected here.

Reply:
I did the changes required by the Reviewer.

Reply:
![Screenshot (239)](https://user-images.githubusercontent.com/106825373/221128884-dfe4d603-c583-486b-a419-10c2852c3fec.png)